server/apps/newsletter/views.py

```
@@ -3,6 +3,12 @@
 from django.db import IntegrityError
 from django.shortcuts import render
 from .models import Subscriber
+from .services import BeehiivService
```

Ruff (0.8.2)

`.services.BeehiivService` imported but unused

Remove unused import: `.services.BeehiivService`

(F401)